### DISCUSSION TITLE
Show beatmaps max combo on result screen

### DISCUSSION THREAD

**Discussion by respektive** [ppy/osu#17303] [Ideas] [answered]
I think it would make sense to see the maximum achievable combo on the result screen, similar to how it shows for slider ends and ticks.
![asdfghh](https://user-images.githubusercontent.com/22381687/158823465-b30990ba-2e62-42f1-bfab-a7e11c715eee.png)

**Comment by frenzibyte** [comment]
The only potential issue with this is that you can have "PERFECT" show up when the combo doesn't match max combo (see https://github.com/ppy/osu/issues/13238).

Would make it look something like this:
![CleanShot 2022-03-18 at 02 38 48](https://user-images.githubusercontent.com/22781491/158911111-836055a1-73ff-4f95-b1b5-3a51c88523f5.png)

Unsure how to feel about that, I guess it's fine for now because there's an issue tracking it?

**Reply by bdach** [reply]
if this is only an issue on stable imported scores as the linked issue says then yes, i would say that fixing this issue is out of scope here

**Reply by frenzibyte** [reply]
I think it also affects lazer scores because large tick misses are not counted, but yeah probably out of scope anyways.

**Reply by peppy** [reply]
Is that fractional display in the flyte-design? I think it looks okay, so am okay with seeing this implemented.

**Reply by frenzibyte** [reply]
Yeah, taken from the pre-existing implementation for "slider ticks" and "slider ends" (all inherit `CounterStatistic` which provides that).

**Reply by bdach** [reply]
> I think it also affects lazer scores because large tick misses are not counted, but yeah probably out of scope anyways.

i don't get this. the max combo count should only include judgements that... affect combo, no?

**Reply by frenzibyte** [reply]
Yeah, and `LargeTickMiss` affects combo:
https://github.com/ppy/osu/blob/a352a140bc145f342fadc300797a2b05ade3f920/osu.Game/Rulesets/Scoring/HitResult.cs#L124-L144

Have pushed a quick fix to that in the PR #17324.

**Comment by frenzibyte** [comment, marked as answer]
Will be resolved by #17324.

**Reply by respektive** [reply]
This works in the tests on master for me, but not actually in-game. Neither on the release from today or by running it from master branch :thinking: 

Edit: tracked here https://github.com/ppy/osu/issues/17347
